commerce_coordinator/apps/frontend_app_payment/views.py

```diff
@@ -57,3 +63,22 @@ def put(self, request):
         params = input_serializer.data
         payment_details = DraftPaymentRequested.run_filter(params)
         return Response(payment_details)
+
+
+class GetActiveOrderView(APIView):
+    """Get Active Order View"""
+    authentication_classes = (JwtAuthentication,)
+    permission_classes = (IsAuthenticated,)
+
+    def get(self, request):
+        """return the user's current active order"""
+        params = {
+            'edx_lms_user_id': request.user.lms_user_id
+        }
+        input_serializer = GetActiveOrderInputSerializer(data=params)
+        input_serializer.is_valid(raise_exception=True)
+        params = input_serializer.data
+        order_data = ActiveOrderRequested.run_filter(params)
+        output_serializer = GetActiveOrderOutputSerializer(data=order_data)
```

Comment:
I think the best place to use `GetActiveOrderOutputSerializer` would be in `GetTitanActiveOrder` PipelineStep. 
This serializer should be responsible to reformat keys to the snake case as well. The idea is to update the format as soon as we get a response from Titan.
This way we would be able to avoid manual data transposition in the following code block in the filter.
```
        # Transpose order data to the values frontend_app_payment expects
        active_order['order_total'] = float(active_order.pop('total'))
        # TODO: Do we still need this waffle flag? Normally from ecommerce
        # and was used from cybersource to stripe migration.
        active_order['enable_stripe_payment_processor'] = True
        active_order['products'] = active_order.pop('lineItems')
        active_order['basket_id'] = active_order.pop('uuid')
  ```
We have an example serializer [here](https://github.com/edx/commerce-coordinator/blob/cb4dafb5e81c5dd7005d8075f220cb8570258982/commerce_coordinator/apps/titan/serializers.py#L20).